website/content/en/status/report-2023-04-2023-06/service-jails.adoc

```diff
@@ -7,8 +7,8 @@ link:https://reviews.freebsd.org/D40371[D40371: automatic service jails: some se
 
 Contact: Alexander Leidinger <netchild@FreeBSD.org>
 
-Service jails are an extension to the man:rc[8] system which allow automatic jailing of rc.d services.
-Service jails inherit the filesystem of the parent host or jail, but use all the other limits of a jail (process visibility, restricted network access, filesystem mounting permissions, sysvipc, ...) by default.
+Service jails extend the man:rc[8] system to allow automatic jailing of rc.d services.
+A service jail inherits the filesystem of the parent host or jail, but uses all other limits of the jail (process visibility, restricted network access, filesystem mounting permissions, sysvipc, ...) by default.
```

Comment:
@netchild please, how is this for you? 

Initially, I had difficulty thinking the first sentence should be plural (Service jails … allow) or singular (an extension … allows).